Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Xiaohong-Deng Is there a library _on the project_ that breaks on `2.4.1`? If it is not, we can proceed to the merge.

If an incompatible library to be used by the project is found, it can be solved simply by switching Ruby versions.